internal/cmd/context/create_test.go

```diff
@@ -13,6 +13,10 @@ import (
 )
 
 func TestCreate(t *testing.T) {
+	// Make sure we don't have any environment variables set that could interfere with the test
+	t.Setenv("HCLOUD_TOKEN", "")
+	t.Setenv("HCLOUD_CONTEXT", "")
```

Comment:
To be on the safe side we could get the env for all variables with prefix `HCLOUD_` and unset them.

Unfortunately Go has no `os.GetEnvMap()`, and `os.Environ()` returns `[]string` with `KEY=VALUE` pairs, so lots of additional effort to parse it and I am not sure its worth it.